README.md

```diff
@@ -201,11 +201,11 @@ data in the source table and update the destination embedding table
 with the new embeddings asynchronously.
 
 [Automate AI embedding with pgai Vectorizer](/docs/vectorizer.md) shows you how
-to implement embeddings in your own data. When you create Vectorizers in a Timescale
-Cloud database, embeddings are automatically created and synchronized in the background.
-On a self-hosted Postgres installation, you use a [Vectorizer
-Worker](/docs/vectorizer-worker.md) to asynchronously processes your
-vectorizers.
+to implement embeddings in your own data. On a self-hosted Postgres
+installation, you use a [Vectorizer Worker](/docs/vectorizer-worker.md) to
+asynchronously processes your vectorizers. When you create Vectorizers in a
+Timescale Cloud database, embeddings are automatically created and synchronized
+in the background. Note: Timescale Cloud only supports embedding with OpenAI.
```

Comment:
In theory you can disable scheduling and run the vectorizer worker with ollama yourself right? While still having the db hosted on cloud?

Reply:
Yes! Do you think that's something that we should suggest to users?

Reply:
I would definitely be up for that. Hosting databases is always a bit tricky because you run the risk of losing data if you do something stupid. But the vectorizer is stateless so if I was building an LLM application I think I'd be fine with such a setup.